### PR TITLE
Remove RelatedPerson and PractitionerRole from bulk data. Remove fix_…

### PR DESCRIPTION
**Pull request by yunwwang**
This pull request contains two set of changes
1) Remove RelatedPerson and PractitionerRole from Multe-Patient tests
2) Remove additional logic in metadata extractor for USCore 3.1.0 and regenerate Goal, Procedure tests

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Internal ticket for this PR:
- [x] Internal ticket links to this PR
- [x] Internal ticket is properly labeled (Community/Program)
- [x] Internal ticket has a justification for its Community/Program label
- [x] Code diff has been reviewed for extraneous/missing code
- [x] Tests are included and test edge cases
- [x] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: generator/uscore/metadata_extractor.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def extract_metadata
         capability_statement_json = capability_statement('server')
         add_metadata_from_ig(metadata, ig_resource)
         add_metadata_from_resources(metadata, capability_statement_json['rest'][0]['resource'])
-        fix_metadata_errors(metadata)
+        # fix_metadata_errors(metadata)
```
Comment (Jammjammjamm) [inline generator/uscore/metadata_extractor.rb:26]: If this isn't needed anymore, just remove it completely rather than commenting it out.

**Review by Jammjammjamm — APPROVED** [review]

File: generator/uscore/metadata_extractor.rb (hunk 1 above)
Comment (arscan) [reply generator/uscore/metadata_extractor.rb:26]: Yeah I understand why it is nice to have it in here but commented out (an easy place to know where you are supposed to add metadata errors in case they pop up again), but I think we need to trust that we can figure out how to readd this if others show up without having to have this cluttering things up in the meantime.  I vote to remove this comment and the method.

**Review by arscan — APPROVED** [review]
